src/lib/components/sparkline/SparkLine.component.js

```diff
@@ -0,0 +1,129 @@
+//@flow
+import React from "react";
+import VegaChart from "../vegachart/VegaChart.component.js";
+import { lighten } from "polished";
+type Props = {
+  id: string,
+  title?: string,
+  width?: number,
+  height?: number,
+  data: Array<Object>,
+  xAxis: Object,
+  yAxis: Object,
+  row: Object,
+  color?: string,
+  maxRuleColor?: string,
+  minRuleColor?: string
```

Comment:
you need add docs or comment to explain what the props are

Reply:
Rename with understandable name ;)